boot/nuttx/src/flash_map_backend/flash_map_backend.c

```diff
@@ -801,5 +801,12 @@ int flash_area_id_from_image_offset(uint32_t offset)
 
   BOOT_LOG_INF("offset:%" PRIu32, offset);
 
-  return dev->fa_cfg->fa_id;
+  if (dev != NULL)
+    {
+	  return dev->fa_cfg->fa_id;
```

Comment:
```suggestion
        return dev->fa_cfg->fa_id;
```
Whitespaces instead of tabs.

Reply:
Done